Tests/DatadogTests/Datadog/LoggerTests.swift

```diff
@@ -28,10 +28,10 @@ class LoggerTests: XCTestCase {
         super.tearDown()
     }
 
-    /// Creates `RUMMonitor` instance for tests.
+    /// Creates `Monitor` instance for tests.
     /// The only difference vs. `RUMMonitor.initialize()` is that we disable RUM view updates sampling to get deterministic behaviour.
-    private func createTestableRUMMonitor(configuration: RUMConfiguration = .mockAny()) throws -> DDRUMMonitor {
-        let monitor = RUMMonitor(
+    private func createTestableRUMMonitor(configuration: RUMConfiguration = .mockAny()) throws -> Monitor {
+        let monitor = Monitor(
```

Comment:
Why not RUMMonitor? I don't mind dropping RUM from names. In the end they are `DatadogRUM.Type`, but IMO we should be consistent.

Reply:
Here we refer to `DatadogRUM.Monitor` which is internal type. The `DatadogRUM.RUMMonitor` is already reserved for appearing in the public API.
```swift
// in DatadogRUM:

public protocol RUMMonitorProtocol {}

public class RUMMonitor {
   func shared() -> RUMMonitorProtocol {
      return allGood ? Monitor() : NOPMonitor()
   }
}

internal class Monitor: RUMMonitorProtocol {}
internal class NOPMonitor: RUMMonitorProtocol {}
```
Because we can't have both, we need to chose. What I propose here is to promote `RUMMonitor` in public name as this is what our users are familiar with from V1. Does it make sense given this context 🤔?

Reply:
ah ok! Makes sense now. I thought there's some deeper reason, just couldn't get it without this additional context. Thanks!